BraintreeModels/src/main/java/com/braintreepayments/api/PaymentMethodNonce.java

```diff
@@ -1,186 +1,29 @@
 package com.braintreepayments.api;
 
-import android.os.Parcel;
 import android.os.Parcelable;
-import androidx.annotation.CallSuper;
-import androidx.annotation.Nullable;
 
-import org.json.JSONArray;
-import org.json.JSONException;
-import org.json.JSONObject;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-
-/**
- * Base class representing a method of payment for a customer. {@link PaymentMethodNonce} represents the
- * common interface of all payment method nonces, and can be handled by a server interchangeably.
- */
-public abstract class PaymentMethodNonce implements Parcelable {
-
-    private static final String PAYMENT_METHOD_NONCE_COLLECTION_KEY = "paymentMethods";
-    private static final String PAYMENT_METHOD_TYPE_KEY = "type";
-    private static final String PAYMENT_METHOD_NONCE_KEY = "nonce";
-    private static final String PAYMENT_METHOD_DEFAULT_KEY = "default";
-    private static final String DESCRIPTION_KEY = "description";
-
-    static final String DATA_KEY = "data";
-    static final String TOKEN_KEY = "token";
-
-    protected String mNonce;
-    protected String mDescription;
-    protected boolean mDefault;
-
-    static JSONObject getJsonObjectForType(String apiResourceKey, JSONObject json) throws JSONException {
-        return json.getJSONArray(apiResourceKey).getJSONObject(0);
-    }
-
-    @CallSuper
-    void fromJson(JSONObject json) throws JSONException {
-        mNonce = json.getString(PAYMENT_METHOD_NONCE_KEY);
-        mDescription = json.getString(DESCRIPTION_KEY);
-        mDefault = json.optBoolean(PAYMENT_METHOD_DEFAULT_KEY, false);
-    }
+public interface PaymentMethodNonce extends Parcelable {
```

Comment:
Docstring for this interface?

Reply:
We reverted this back to a base class so the original doc strings apply